### PR TITLE
Grammar and style

### PR DESCRIPTION
**Pull request by heyawhite**
Capitalization of products and various grammar updates to the main README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
+This repository contains example code for an authorship experience that provides feedback as you type using [Perspective API](http://www.perspectiveapi.com/). It is a web app written in
+[Angular2](https://angular.io/)/[TypeScript](https://www.typescriptlang.org/) and uses a [Node Express server](https://expressjs.com/). This project shows how to have a development and a production
+build environment with deployment tools based on [Docker](https://www.docker.com/). You can then deploy to cloud services such as [Google Cloud's App Engine](https://cloud.google.com/appengine/),
+as well as other cloud computing providers.
 
-The app uses the [Perspective API](http://www.perspectiveapi.com/) to score text, via a
-the [perspectiveapi-simple-server](https://github.com/conversationai/perspectiveapi-simple-server).
+The app uses the [Perspective API](http://www.perspectiveapi.com/) to score text, via
+[Simple Server for Perspective API](https://github.com/conversationai/perspectiveapi-simple-server).
```
Comment (rachelrosengoogle) [inline README.md]: I think it's more understandable to call it by the name of the repo (perspectiveapi-simple-server).

Comment (heyawhite) [reply README.md]: I've changed it back

Comment (rachelrosengoogle) [reply README.md]: Thanks! Could you change it back in the other places too?

Comment (heyawhite) [reply README.md]: I don't see it written anywhere else, that should have been the only change.

Comment (rachelrosengoogle) [reply README.md]: I see it on line 27, 37, and 54. Apologies if you already edited this, github's UI for viewing changes can be confusing at times!

Comment (heyawhite) [reply README.md]: It really can be!! I think it should really, actually be fixed this time.

Comment (rachelrosengoogle) [reply README.md]: Thank you!

**Comment by heyawhite** [comment]
@rachelrosengoogle can you merge? I don't have the power to do so.

**Pull request merged**